### ISSUE TITLE
maven build tolerates XSLT processor warnings

### ISSUE THREAD

**Issue by h1alexbel** [objectionary/lints#488]
I made a few changes and ran:

```bash
mvn clean install -Pqulice
```

Getting this:

```text
[INFO] Tests run: 5, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 2.034 s -- in org.eolang.lints.LtInconsistentArgsTest
[INFO] Running org.eolang.lints.LtByXslTest
Warning at xsl:stylesheet on line 6 column 129
  SXWN9019: Stylesheet module file:/org/eolang/parser/_specials.xsl is included or imported
  more than once. This is permitted, but may lead to errors or unexpected behavior
Warning at xsl:stylesheet on line 6 column 129
  SXWN9019: Stylesheet module file:/org/eolang/parser/_specials.xsl is included or imported
  more than once. This is permitted, but may lead to errors or unexpected behavior
Warning at xsl:stylesheet on line 6 column 129
  SXWN9019: Stylesheet module file:/org/eolang/parser/_specials.xsl is included or imported
  more than once. This is permitted, but may lead to errors or unexpected behavior
Warning at xsl:stylesheet on line 6 column 129
...
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  03:14 min
[INFO] Finished at: 2025-04-15T09:42:35+02:00
[INFO] ------------------------------------------------------------------------
```

Even though my `.xsl` file is broken, the build is still clean. Instead, I suggest to fail the build, if any XSLT warnings being propagated during the run.

**Comment by h1alexbel** [comment]
@yegor256 WDYT>

**Comment by yegor256** [comment]
@h1alexbel you are right, but the problem is deeper, in the `jcabi-xml` module: https://github.com/jcabi/jcabi-xml/blob/master/src/main/java/com/jcabi/xml/XSLDocument.java#L405-L436 As you see, we collect errors (and warnings!) into the `errors` variable, but then we don't throw in warnings are there. We only print them to the log. I suggest you report a bug to `jcabi-xml`.
